Services/source/housekeeping/housekeeping_service.c

```diff
@@ -303,6 +309,42 @@ Found_file exists(const char *filename){
     return FILE_NOT_EXIST;
 }
 
+Result store_config() {
+  int32_t fout = red_open(hk_config, RED_O_CREAT | RED_O_RDWR); //open or create file to write binary
+  if (fout == -1) {
+    ex2_log("Failed to open or create file to write: '%s'\n", hk_config);
+    return FAILURE;
+  }
+  red_write(fout, &MAX_FILES, sizeof(MAX_FILES));
+  red_write(fout, &current_file, sizeof(current_file));
```

Comment:
Could you verify that sizeof() is correct here? The parameter sounds like it's taking the size of a file rather than a structure

Reply:
current_file refers to what file number data will go into when the program saves housekeeping data. it's a uint8_t. In this situation it is loaded from a file so the program can tell what file was going to be written to next when housekeeping is collected. it's the basis of how the service keeps everything organized and in order. it does seem like a file pointer. but it's just a file counter for the numbers used in file names.

Reply:
Ah, makes sense